### ISSUE TITLE
Any hope to see tldp.org web pages updated?

### ISSUE THREAD

**Issue by fpl** [tLDP/LDP#112]
It seems that the website and all its mirrors have been frozen for years, which is unfortunate because it is still among the main results after googling about Linux and its tools. Any hope to see the whole site revamped?

**Comment by ser** [comment]
yes it seems it got blocked for a while recently, i am working on restoring the automatisation
